website/content/en/docs/cli/operator-sdk_generate_bundle.md

```diff
@@ -30,62 +38,57 @@ operator-sdk generate bundle [flags]
-  $ make bundle
-  $ export USERNAME=<quay-namespace>
-  $ export BUNDLE_IMG=quay.io/$USERNAME/memcached-operator-bundle:v0.0.1
-  $ make bundle-build BUNDLE_IMG=$BUNDLE_IMG
-
-  # The above recipe runs the following commands manually. First it creates bundle
-  # manifests, metadata, and a bundle.Dockerfile:
-  $ make manifests
-  /home/user/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
-  $ operator-sdk generate kustomize manifests
-
-  Display name for the operator (required):
-  > memcached-operator
-  ...
-
+  # If running within a project or in a project that uses kustomize to generate manifests,
+	# make sure a kustomize directory exists that looks like the following 'config/manifests' directory:
   $ tree config/manifests
   config/manifests
   ├── bases
   │   └── memcached-operator.clusterserviceversion.yaml
   └── kustomization.yaml
-  $ kustomize build config/manifests | operator-sdk generate bundle --overwrite --version 0.0.1
-  Generating bundle manifest version 0.0.1
+
+  # Generate a 0.0.1 bundle by passing manifests to stdin:
+  $ kustomize build config/manifests | operator-sdk generate bundle --version 0.0.1
+  Generating bundle version 0.0.1
   ...
 
-  # After running the above commands, you should see this directory structure:
-  $ tree bundle
-  bundle
+  # If running outside of a project or in a project that does not use kustomize to generate manifests,
+	# make sure cluster-ready manifests are available on disk:
+  $ tree deploy/
+  deploy/
+  ├── crds
+  │   └── cache.my.domain_memcacheds.yaml
+  ├── deployment.yaml
+  ├── role.yaml
+  ├── role_binding.yaml
+  ├── service_account.yaml
+  └── webhooks.yaml
+
+  # Generate a 0.0.1 bundle by passing manifests by dir:
+  $ operator-sdk generate bundle --input-dir deploy --version 0.0.1
+  Generating bundle version 0.0.1
+  ...
+
+  # After running in either of the above modes, you should see this directory structure:
+  $ tree bundle/
+  bundle/
   ├── manifests
   │   ├── cache.my.domain_memcacheds.yaml
   │   └── memcached-operator.clusterserviceversion.yaml
   └── metadata
       └── annotations.yaml
 
-  # Then it validates your bundle files and builds your bundle image:
-  $ operator-sdk bundle validate ./bundle
-  $ docker build -f bundle.Dockerfile -t $BUNDLE_IMG .
-  Sending build context to Docker daemon  42.33MB
-  Step 1/9 : FROM scratch
-  ...
-
-  # You can then push your bundle image:
-  $ make docker-push IMG=$BUNDLE_IMG
-
 ```
 
 ### Options
 
 ```
       --channels string          A comma-separated list of channels the bundle belongs to (default "alpha")
-      --crds-dir string          Root directory for CustomResoureDefinition manifests
+      --crds-dir string          Directory to read cluster-ready CustomResoureDefinition manifests from. This option can only be used if --deploy-dir is set
```

Comment:
hmm, i guess none of them are required there's a compatability-matrix thing going on. oh well